src/environmentd/src/http/webhook.rs

```diff
@@ -81,6 +84,7 @@ pub async fn handle_webhook(
 
 /// Append the provided `body` and `headers` to the webhook source identified via `database`,
 /// `schema`, and `name`.
+#[instrument(level = "debug", target = "webhook")]
```

Comment:
do we actually want to override the target here? it makes it different to configure with filters